src/osd/OSD.cc

```diff
@@ -5312,7 +5313,10 @@ void OSD::_preboot(epoch_t oldest, epoch_t newest)
 	   << oldest << ".." << newest << dendl;
 
   // ensure our local fullness awareness is accurate
-  heartbeat();
+  {
+    Mutex::Locker l(heartbeat_lock);
```

Comment:
@liewegas need review. std::lock_guard is not being recognized in mimic.

Reply:
@pdvian In addition to "resolved for <function name>" in the Conflicts section, could you also say something like "std::lock_guard is not being recognized in mimic; used Mutex::Locker instead"?